.CI/create_feedstocks.py

```diff
@@ -173,17 +150,8 @@ def write_token(name, token):
             else:
                 subprocess.check_call(['conda', 'smithy', 'register-github', feedstock_dir] + owner_info)
```

Comment:
Adding `--add-teams` here does the same thing. I think it's better to have it in this repo because the webservice sometimes fails and trying this here makes sure the team is created.

Reply:
Yep, we can certainly add that flag here. That was my initial plan when adding that flag to `conda-smithy`.

Though would like to see the webservice get more robust. Had noticed issues began with the linter some time back when PR ( https://github.com/conda-forge/conda-forge-webservices/pull/76 ) went in. Expect there is something off with that change to the webservices, but never narrowed down what.

Reply:
Okay, then let's wait till the new conda-smithy  release because I fixed a bug there in adding teams

Reply:
I think https://github.com/conda-forge/conda-forge-webservices/commit/e14f009d54829d3b6d97fed829245dd65dc43f4c is what is causing the issues. In this, `remote_repo.get_pull` is failing with 404. Maybe the webhook event is sent before the Github API is updated to reflect the new PR.

Reply:
Hmm...that's disappointing. Were you seeing this in the Heroku logs?

Had thought about adding a small timeout and retrying some fixed number of times (maybe 3-5). What do you think?

Reply:
Sorry for the digression, we can revisit that at the webservices repo.

Have added the `--add-teams` flag here.

Reply:
Moved the call to `register-github` with the `--add-teams` flag later on to make sure the feedstock has already been created and is fully functional before giving out permissions.